server/src/internalClusterTest/java/org/opensearch/action/admin/cluster/node/tasks/AbstractTasksIT.java

```diff
@@ -110,6 +110,7 @@ protected int numberOfEvents(String actionMasks, Function<Tuple<Boolean, TaskInf
 
     /**
      * Returns all events that satisfy the criteria across all nodes
+     * TODO : This javadoc is wrong. it's a copy paste from numberOfEvents (the method above)
```

Comment:
We should probably remove that and fix the comment

Reply:
fixed in new commit